packages/angular_devkit/build_angular/src/extract-i18n/index.ts

```diff
@@ -29,6 +31,8 @@ import { Format, Schema } from './schema';
 
 export type ExtractI18nBuilderOptions = Schema & JsonObject;
 
+type LocalizeMessage = import('@angular/localize').ɵParsedMessage;
```

Comment:
My goal here was to prevent accidental eager loading of a value from `@angular/localize` since it's an optional peer dependency.  I'll switch to an `import type` statement which provides both benefits.